modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ImportTranslationResultsDisplayContext.java

```diff
@@ -47,6 +47,56 @@ public ImportTranslationResultsDisplayContext(
 		_uploadedFileName = uploadedFileName;
 	}
 
+	public int getImportErrorsCount() {
+		return _importErrors.size();
+	}
+
+	public Map<String, String> getImportErrorsMap() {
+		return _importErrors;
+	}
+
+	public int getImportSuccessCount() {
+		return _importSuccess.size();
+	}
+
+	public List<String> getImportSuccessEntries() {
+		return _importSuccess;
+	}
+
+	public String getImportTranslationURL() {
+		return PortletURLBuilder.createRenderURL(
+			_liferayPortletResponse
+		).setMVCRenderCommandName(
+			"/translation/import_translation"
+		).setRedirect(
+			this::getRedirect
```

Comment:
Is this ok? why are you passing a method reference instead of the value returned by getRedirect()?